torch/distributed/_sharded_tensor/__init__.py

```diff
@@ -0,0 +1,41 @@
+from torch.distributed._sharding_spec import ShardingSpec
+from .api import (
+    ShardedTensor,
+)
+
+def empty(
+        sharding_spec: ShardingSpec,
+        *size,
+        dtype=None,
```

Comment:
You can add an internal function `init` that requires an `init_meth` besides all the args here, where `init_meth` can be `torch.empty`, `torch.zeros`, or `torch.ones`. This way it will be trivial to support zeroes and ones on the sharded tensor.

Reply:
I think we can refactor this once we need to add more functionality. Basically in the PR where we add such additional methods, it would be easier to properly refactor this.

Reply:
Sure. That's why I also label this comment as optional. Such refactoring can easily be done in the future on demand.